creator-node/src/routes/healthCheck.js

```diff
@@ -124,6 +124,18 @@ module.exports = function (app) {
       return successResponse(resp)
     }
   }))
+
+  app.get('/ip_check', handleResponse(async (req, res) => {
```

Comment:
example:

Req:
```
GET http://<remote ip>:4000/ip_check (cn1 with openresty enabled)
Headers: X-Forwarded-For: <some spoofed ip>
```
Resp: 
```
{
    "data": {
        "ip": "<my ip>"
    },
    "signer": "0x617601de92b450b7a07b4804fcd2a43d837bbb70",
    "timestamp": "2021-12-08T21:20:48.637Z",
    "signature": "0xa4c93c6f7d6653b6d719216f4732821800e88bcee06a0aefbf6ef4ebea553c6f2de8d9df0114066a8885992b6f5eccf9a58274db2d9e4250261a0cf40c3bf7ca1b"
}
```

Reply:
cc @dmanjunath

Reply:
change in nginx conf https://github.com/AudiusProject/audius-protocol/pull/2131/files#diff-4a39dd46c9418f69f3b7d6f4a8cbc8a0ce2d70df15790923ea75a7c8a31e9a4cR22